opal/mca/common/cuda/common_cuda.c

```diff
@@ -300,6 +303,15 @@ void mca_common_cuda_register_mca_variables(void)
                                  MCA_BASE_VAR_SCOPE_READONLY,
                                  &mca_common_cuda_cumemcpy_timing);
 #endif /* OPAL_ENABLE_DEBUG */
+
+#if OPAL_CUDA_GPU_MEM_WAR
+    (void) mca_base_var_register("ompi", "mpi", "common_cuda", "gpu_mem_check_war",
+                                 "Set to 1 to enable GPU memory check war",
+                                 MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
+                                 OPAL_INFO_LVL_9,
```

Comment:
Is this MCA param for a *correctness* issue, or a *performance tuning* issue?

If it's a *correctness* issue, I'd make it level 2 or 3 (per https://github.com/open-mpi/ompi/wiki/MCAParamLevels).
If it's a *tuning* issue, I'd make it level 4 or 5.